src/main/scala/cognite/spark/v1/DefaultSource.scala

```diff
@@ -165,6 +165,26 @@ class DefaultSource
       val relation = new AssetHierarchyBuilder(config)(sqlContext)
       relation.buildFromDf(data)
       relation
+    } else if (resourceType == "datapoints" || resourceType == "stringdatapoints") {
+      val relation = resourceType match {
+        case "datapoints" =>
+          new NumericDataPointsRelationV1(config)(sqlContext)
+        case "stringdatapoints" =>
+          new StringDataPointsRelationV1(config)(sqlContext)
+      }
+      if (config.onConflict == OnConflict.Delete) {
+        // Datapoints support 100_000 per request when inserting, but only 10_000 when deleting
```

Comment:
Surprising.. wonder why!

Reply:
I guess because it's deleted by ranges, so it allows at most 10k ranges... not sure however, I guess we just have to accept the limitation